### PR TITLE
Improve Github Actions (BFF-1423) 

### PR DESCRIPTION
**Pull request by MaikelStuivenberg**
## Description
Automatic assign labels, template for release notes + optimising versioning etc :)

### CODE REVIEW

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -52,9 +49,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Ruby
-        uses: ruby/setup-ruby@v1.118.0
+        uses: ruby/setup-ruby@v1
         with: 
-          ruby-version: ${{ inputs.ruby-version }}
```
Comment (MaikelStuivenberg) [inline .github/workflows/android.yml:57]: this is not needed as we have a .ruby-version file

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -52,9 +49,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Ruby
-        uses: ruby/setup-ruby@v1.118.0
+        uses: ruby/setup-ruby@v1
```
Comment (MaikelStuivenberg) [inline .github/workflows/android.yml:52]: only v1 is recommended by the plugin

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -38,8 +38,7 @@ jobs:
       MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
       LOGIT_API_KEY: ${{ secrets.LOGIT_API_KEY }}
     with:
-      flutter-version: '3.13.7'
-      ruby-version: '2.6.0'
+      flutter-version: '3.13.x'
```
Comment (MaikelStuivenberg) [inline .github/workflows/main.yaml:41]: In this way we allow bugfixes

**Review by alexbejann — APPROVED** [review]

File: .github/workflows/android.yml (hunk 1 above)
Comment (jul-reb) [reply .github/workflows/android.yml:57]: That is not how it works sadly!
The default ruby installer in github is not aware of the .ruby-version file.

Comment (MaikelStuivenberg) [reply .github/workflows/android.yml:57]: According to the docs it is

Comment (jul-reb) [reply .github/workflows/android.yml:57]: nvm!

**Pull request merged**